docs/testing.md

```diff
@@ -125,6 +126,9 @@ and has three fields:
   `"pass"`, defining the information needed to talk to the IPMI
```

Comment:
Line 119 is in fact correct English.

The ipmi thing is out of date; when we made the OBM drivers pluggable this changed. So I just updated that to describe the current behavior.

I'm not sure what you mean when you say "use the format of `<switches>`"